cob_bringup/drivers/sick_lms1xx.launch

```diff
@@ -6,11 +6,11 @@
 	<arg name="name"/>
 
 	<!-- load parameter -->
-	<rosparam ns="$(arg name)" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
+	<rosparam ns="$(arg name)/driver" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
```

Comment:
Yes, the difference is that lms1xx uses a private NodeHandle to retrieve the parameters while s300 doesn't!
In order to make it consistent, I'd prefer to modify the cob_driver/sick_lms1xx to use a non-private NodeHandle, too!